resources/kcp/values.yaml

```
@@ -18,7 +18,7 @@ global:
       version: "PR-723"
```

if you're bumping this base image: https://github.com/kyma-project/control-plane/blob/b267affed5890860795538b9be57ee6c7b4cd6e4/components/kyma-environment-broker/Dockerfile.cleanup

You'd like to bump the cleanup job too.

```
Dockerfile.sac -> kyma_environments_subaccount_cleanup_job
Dockerfile.cleanup -> kyma_environments_cleanup_job
```

```suggestion
      version: "PR-857"
```

---

thanks!